### ISSUE TITLE
Row classes get removed on cell value change 

### ISSUE THREAD

**Issue by alvarotrigo** [handsontable/handsontable#3180]
[Reproduction online](http://jsfiddle.net/7u1kxjLe/18/)
- Click "Validate" to add the class
- Then click "Click to change value" to update a cell value in that same row

Manually adding a class to any row will result in the deletion of that class when modifying any cell element by calling `setDataAtRowProp`.

In my case, I want all the rows to get the same color (thus the class I want to add to the `tr` elements) when I'm about to call a resful API to validate their content. 

I know using the cell renderers you can get something similar, but would rather not have to add a class per cell, but per row, as this will be much faster and the result can be the same by using the proper CSS. `tr.myClass td { background: red; }` 

Basically:

``` javascript
//add 1st row class 
$('#validate').click(function(){
    var tbody = $('#example1')
            .find('table').first().find('tbody').first();

    tbody.find('tr').eq(0).addClass('jobs-style');
});

//removing any previously added class
$('#changeValue').click(function(){
    hot1.setDataAtRowProp(0, 'phone', 'Demo!!');
});
```

**Comment by AMBudnik** [comment]
Hi @alvarotrigo please take a look at this updated example http://jsfiddle.net/7u1kxjLe/19/

While rendering table takes cells' meta from a `className` that we changed in `55` line so the class isn't erased after updating cell's value by a <kbd>Validate</kbd> button.

**Comment by alvarotrigo** [comment]
Thanks for the answer.
Seems like a possible workaround, but it still setting the class for each cell of the table, not for each row. That way if the table is 10*100 it will have to perform 1000 operations instead of 100.

There's another issue there anyway: I tried to look into `setCellMeta`, but the documentation doesn't specify which possible `key` values there can be (again).

**Comment by AMBudnik** [comment]
I know what you mean - unfortunately currently it's the only way.
I'm marking this issue as a proposition to update docs about `setCellMeta`.

**Comment by swistach** [comment]
It's the same issue as #2377.
